var/spack/repos/builtin/packages/openfoam-com/package.py

```diff
@@ -258,12 +258,14 @@ class OpenfoamCom(Package):
     """
 
     homepage = "http://www.openfoam.com/"
-    baseurl  = "https://sourceforge.net/projects/openfoamplus/files"
+    baseurl  = "https://sourceforge.net/projects/openfoamplus/files/"
+    gitrepo =  "https://develop.openfoam.com/Development/OpenFOAM-plus.git"
 
+    version('1706', '630d30770f7b54d6809efbf94b7d7c8f',
+            url=baseurl + 'v1706/OpenFOAM-v1706.tgz')
     version('1612', 'ca02c491369150ab127cbb88ec60fbdf',
-            url=baseurl + '/v1612+/OpenFOAM-v1612+.tgz')
-    version('plus', branch='develop',  # Note: needs user credentials
-            git='https://develop.openfoam.com/Development/OpenFOAM-plus.git')
+            url=baseurl + 'v1612+/OpenFOAM-v1612+.tgz')
+    version('plus', branch='develop', git=gitrepo)  # Needs credentials
```

Comment:
Thanks for the info. I didn't know that `develop` received any special treatment. Now that I know this, I'm really of two minds. It is naturally great to provide `develop` as the newest, but I'm not sure it this is the wisest for the normal cluster (ie, prefer stable vs. develop). However it seems that default version (eg, spack spec -I openfoam-com) is always the newest numeric, which would be ok. Can you confirm this behaviour?